ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
> I think we'll remove this model at some point (now?), so this import will break.

I don't think we should remove it now, it's what enables the migration.

We can remove it at some point in the future, sure, but... is there any rush?

The fact there are very few doesn't mean we shouldn't try and do the right thing. It's done, it works, it didn't take much time developing, and it would take more time removing it.

When we do remove the model in the future, we can just guard the `import` in a `try`/`except ImportError` block, so that the migration doesn't do anything when the model doesn't exist any more.